styles/styles.css

```diff
@@ -0,0 +1,239 @@
+    flex-direction: row;
+    justify-content: space-between;
+
+    padding: 2rem;
+}
+.case-study-text {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 20em;
+    justify-content: center;
+    row-gap: 1.5rem;
+    img {
+        height: 50px;
+        width: 50px;
+        flex-grow: 0;
+    }
+}
+
+.services-article {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 30%;
+    padding: 1rem;
+
+    border: 1px solid var(--color-purple);
+    img {
+        height: 50px;
+        width: 50px;
+        flex-grow: 0;
+    }
+    h2 {
+        color: var(--color-purple);
+    }
+    p {
+        color: var(--textcolor-article-grey);
+        font-size: 12px;
+    }
+}
+/*Area Specific*/
+    /*header*/
+        .introblock {
+            padding-top: 50px;
+            text-align: center;
+
+            button {
+                background-color: var(--color-pink);
+                width: fit-content;
+                padding: 20px;
+                border-radius: 25px;
+                color: white;
+            }
+        }
+        .inner-container-large-nav {
+            display: flex;
+            flex-direction: column;
+            width: 95%;
+            align-items: flex-end;
+            justify-content: center;
+        }
+    /* Main */
+        /* section 1 (introduction) */
+
+            #question-intro {
+                padding: 30px;
+            }
+            .intro-inner-container {
+                display: flex;
+                flex-direction: column;
+                flex-basis: 30rem;
+                justify-content: center;
+                row-gap: 1.5rem;
+                text-align: center;
+            }
+        /* section 2 (case studies) */
+
+            .casestudies {
+                background-color: var(--bg-color-article-lightgrey);
+            }
+
+            #reverse {
+                flex-direction: row-reverse;
+            }
+        /* Section 3 (Services) */
+            .inner-container-services {
+                display: flex;
+                flex-direction: row;
+                flex-wrap: wrap;
+
+                width: 95%;
+                align-items: center;
+
+                justify-content: space-between;
+                row-gap: 1.5rem;
+                padding-top: 3rem;
+                padding-bottom: 3rem;
+            }
+        /* Section 4 (Form) */
+            .form-section {
+                background-color: var(--color-blue);
+                color: white;
+                padding-top: 3rem;
+                padding-bottom: 3rem;
+
+                button {
+                    background-color: var(--color-pink);
+                    padding: 20px;
+                    border-radius: 25px;
+                    width: 10rem;
+                    margin-top: 1rem;
+                    color: white;
+                }
+            }
+/* Media queries */
+
+@media screen and (min-width: 768px) {
+    body {
+        font-size: 14px;
```

Comment:
De ene keer gebruik je px en de andere keer rem. Probeer hier consistenter in te zijn.